src/audio_to_image_model/model_audio_to_image.py

```diff
@@ -97,29 +131,51 @@ def _test(dataloader, model, loss_fn):
     print("Running on device: {}".format(device))
 
     # Initialize the model
-    model = CustomCNN(num_classes).to(device)
+    model = PretrainedEfficientNetBirdClassifier(num_classes).to(device)
     print(model)
+    number_trainable_params(model)
 
     # Initialize the model weights
     model.apply(init_weights)
 
     # Define the loss function and the optimizer
-    criterion = nn.BCELoss()
+    # criterion = nn.BCELoss()
+    # FIXME: New loss does not need Sigmoid/softmax?
+    criterion = nn.CrossEntropyLoss()
+
     optimizer = optim.Adam(model.parameters(), lr=0.001)
 
     # Train the model
-    num_epochs = 10
+    num_epochs = 20
+    train_losses = []
+    train_accuracies = []
+    val_losses = []
+    val_accuracies = []
     for epoch in range(num_epochs):
         # Train
-        train(train_loader, model, criterion, optimizer, epoch +1, num_epochs)
+        train_loss, train_accuracy = train(train_loader, model, criterion, optimizer, epoch + 1, num_epochs)
+        train_losses.append(train_loss)
+        train_accuracies.append(train_accuracy)
+
         # Evaluate on the validation set
-        _test(val_loader, model, criterion)
+        val_loss, val_accuracy = _test(val_loader, model, criterion)
+        val_losses.append(val_loss)
+        val_accuracies.append(val_accuracy)
 
-    # TODO: Save model not only for inference but also for training
-    torch.save(model.state_dict(), "../../models/model.pth")
+    # Save the model
+    torch.save(model.state_dict(), "../../models/model2.pth")
```

Comment:
PLEASE change this ;)
We should not start with naming like model, model2 and end up with FINAL_modell_v123
Also something were ClearML could help

Reply:
I named the model in my code e.g. audio_1d_CNN_model

Reply:
> PLEASE change this ;) We should not start with naming like model, model2 and end up with FINAL_modell_v123 Also something were ClearML could help

Why not 🙃 xD? Changed it